src/foundry/foundry.js/applications/formApplications/keybindingsConfig.d.ts

```diff
@@ -103,6 +113,88 @@ declare class KeybindingsConfig<
    */
   protected _onClickCategoryCollapse(event: MouseEvent): void;
 
+  /**
+   * Handle left-click events to reset all Actions to Default
+   * @internal
+   */
+  protected _onClickResetActions(event: MouseEvent): Promise<void>;
```

Comment:
I think we can just make the `_on...` handlers return `void`. Doesn't matter much, though.